.github/workflows/e2e.yml

```diff
@@ -39,19 +39,18 @@ jobs:
         if: ${{ matrix.os == 'ubuntu-latest' && matrix.head == 'headful' }}
         timeout-minutes: 15
         run: >
-          VERBOSE=true
-          DEBUG=*
-          CHROME_BIN="${{ env.chromium_binary }}"
           xvfb-run --auto-servernum
           npm run e2e-${{ matrix.head }}
+        # For verbose logging, set `DEBUG: 'bidiMapper:mapperDebug:*'` and `VERBOSE: true`.
+        env:
+          CHROME_BIN: ${{ env.chromium_binary }}
       - name: Run E2E tests
         if: ${{ matrix.os == 'macos-latest' || (matrix.os == 'ubuntu-latest' && matrix.head == 'headless') }}
         timeout-minutes: 15
-        run: >
-          VERBOSE=true
-          DEBUG=*
```

Comment:
Removed `DEBUG` and `VERBOSE` because they are EXTREMELY noise, and makes it VERY difficult to visualize the logs (~40k+ lines of output in the github logs). Let's enable this on-demand only, when needed.